src/bundle/Resources/views/themes/standard/twig_components/script.html.twig

```diff
@@ -1,8 +1,19 @@
 <script
         type="{{ type }}"
         src="{{ src }}"
-        {{ async ? 'async="' ~ async ~ '"' : "" }}
-        {{ defer ? 'defer="' ~ defer ~ '"' : "" }}
-        {{ crossorigin ? 'crossorigin="' ~ crossorigin ~ '"' : "" }}
-        {{ integrity ? 'integrity="' ~ integrity ~ '"' : "" }}
+        {% if async is not empty %}
+            async="{{ async }}"
+        {% endif %}
+
+        {% if defer is not empty %}
+            defer="{{ defer }}"
+        {% endif %}
+
+        {% if crossorigin is not empty %}
+            crossorigin="{{ crossorigin }}"
+        {% endif %}
+
+        {% if integrity is not empty %}
+            integrity="{{ integrity }}"
+        {% endif %}
```

Comment:
While this is great for readability, consider making this "spaceless" so that we're not generating whitespace noise in output.

While response is usually compressed, it still will be uncompressed for the browser, and that is just wasteful :)

```suggestion
        {%- if async is not empty %}
            async="{{ async }}"
        {% endif -%}

        {%- if defer is not empty %}
            defer="{{ defer }}"
        {% endif -%}

        {%- if crossorigin is not empty %}
            crossorigin="{{ crossorigin }}"
        {% endif -%}

        {%- if integrity is not empty %}
            integrity="{{ integrity }}"
        {% endif -%}
```

Please, if you do use this suggestion, ensure that there is a single space separating tag attributes.